services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_tasks.py

```diff
@@ -246,6 +288,8 @@ async def _generate_tasks_list_from_project(
             last_heartbeat=None,
             created=arrow.utcnow().datetime,
             modified=arrow.utcnow().datetime,
+            pricing_info=pricing_info.dict() if pricing_info else None,
+            hardware_info=hardware_info.dict() if hardware_info else None,
```

Comment:
Q: dynamic services do not require these?

Reply:
That was done already in the PR yesterday and it is already in the master: https://github.com/ITISFoundation/osparc-simcore/pull/4869